README.md

```diff
@@ -265,6 +265,61 @@ The static site that gets built is pushed to the `documentation` branch of this
 We host the site on netlify. On master branch builds (see `.github/workflows/documentation.yml`), we push the built docs to
 the `documentation` branch. Netlify automatically re-deploys the docs pages whenever there is a change to that branch.
 
+## Releases
+### Release Timeline
+**For Rasa Open Source, we usually commit to time-based releases, specifically on a monthly basis.** 
+This means that we commit beforehand to releasing a specific version of Rasa Open Source on a specific day, 
+and we cannot be 100% sure what will go in a release, because certain features may not be ready. 
+
+At the beginning of each quarter, the Rasa team will review the scheduled release dates for all products and make sure 
+they work for the projected work we have planned for the quarter, as well as work well across products.
+
+**Once the dates are settled upon, we update the respective milestones.**
+
+### Cutting a Major / Minor release
+#### A week before release day
+
+1. **Make sure the [milestone](https://github.com/RasaHQ/rasa/milestones) already exists and is scheduled for the 
+correct date**
+2. **Take a look at the issues & PRs that are in the milestone**: does it look about right for the release highlights 
+we are planning to ship? Does it look like anything is missing? (don't worry about being aware of every PR that should 
+be in, but it's useful to take a moment to evaluate what's assigned to the milestone)
+3. **Post a message on the engineering Slack channel**, letting the team know you'll be the one cutting the upcoming 
+release, as well as:
+    1. Providing the link to the appropriate milestone
+    2. Reminding everyone to go over their issues and PRs and please assign them to the milestone
+    3. Clarifying the scheduled date for the release
+
+#### Release day! 🤘
+
+1. **At the start of the day, post a small message on slack announcing release day!** Communicate you'll be handling 
+the release, and the time you're aiming to start releasing (again, no later than 4pm, as issues may arise and 
+cause delays)
+2. **Go over the milestone and evaluate the status of any PR merging that's happening. Follow up with people on their 
```

Comment:
Again: Seems too late for me: In my opinion this should be done a week before the release and then maybe again 2-3 days to the release.

Reply:
Should we also add something like: "The PR / issue owners are responsible for communicating any issues which might be release relevant."?

Reply:
But again, this is a repetition of what is already asked a week before on these lines, no? Or am I missing something?
https://github.com/RasaHQ/rasa/pull/7266/files/1ff7d8a06968795617ecc4d6df933ca1cd47742c#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R284-R291

Reply:
> Should we also add something like: "The PR / issue owners are responsible for communicating any issues which might be release relevant."?

I can add this, of course 😄

Reply:
> I can add this, of course 😄

seems you've already added it 👍 

What about https://github.com/RasaHQ/rasa/pull/7266/files#r522968715 ?

Reply:
> What about https://github.com/RasaHQ/rasa/pull/7266/files#r522968715 ?

That was what I was referring to [here](https://github.com/RasaHQ/rasa/pull/7266/files/1ff7d8a06968795617ecc4d6df933ca1cd47742c#r525030529) 😬  sorrt if that wasn't too clear 😅

Reply:
Ah, sorry. Thought this relates to `The PR / issue owners are responsible for communicating any issues which might be release relevant.` part. 

I agree that it's somewhat duplicate. But can't we move points 1-2 to at least a day before? As release captain I don't wanna bother with these items on the release time. Also this approach doesn't work if a PR owner is e.g. in the United States and doesn't see the message until late afternoon European time.

Reply:
Ok I'll move this to `the day before release` 🚀